package gunir;

message InputInfo {
    required string table_name = 1;
    required string schema_descriptor = 2;
    required string tablet_file_prefix = 3;
}
